tests/modules/krona/ktimporttaxonomy/main.nf

```diff
@@ -2,15 +2,27 @@
 
 nextflow.enable.dsl = 2
 
-include { KRONA_KTIMPORTTAXONOMY } from '../../../../modules/krona/ktimporttaxonomy/main.nf'
+include { KRONA_KTIMPORTTAXONOMY as KRONA_TAXONOMY_READS  } from '../../../../modules/krona/ktimporttaxonomy/main.nf'
+include { KRONA_KTIMPORTTAXONOMY as KRONA_TAXONOMY_REPORT } from '../../../../modules/krona/ktimporttaxonomy/main.nf'
```

Comment:
I would rather imporat as:

`KRONA_KTIMPORTTAXONOMY_READS`
`KRONA_KTIMPORTTAXONOMY_REPORT`

I got very confused what I was looking at in the tests...

Reply:
agreed